openlibrary/templates/type/i18n_page/edit.html

```diff
@@ -25,14 +25,14 @@ <h1>$page.key</h1>
 <div id="contentBody">
     $:macros.TypeChanger(page.type)
     <div class="formElement">
-        <div class="label"><label for="_lang">Available Translations:</label></div>
+        <div class="label"><label for="_lang">$_("Available Translations:")</label></div>
         <div class="input">
             <select name="_lang" onchange="window.location = (window.location.protocol + '//' + window.location.host + this.options[this.selectedIndex].value + window.location.search + '&redirect=$page.key')" id="lang_dropdown">
-            <option value="">Please select one ...</option>
-            <option value="${key}.en">English (en)</option>
-            <option value="${key}.it">Italian (it)</option>
-            <option value="${key}.es">Spanish (es)</option>
-            <option value="${key}.te">Telugu (te)</option>
+            <option value="">$_("Please select one ...")</option>
+            <option value="${key}.en">$('English (en)')</option>
+            <option value="${key}.it">$('Italian (it)')</option>
+            <option value="${key}.es">$('Spanish (es)')</option>
+            <option value="${key}.te">$('Telugu (te)')</option>
```

Comment:
Whatever this dropdown is listing languages for seems very out-of-date. Is it unused?

Reply:
@hornc Hmm very likely! I'll do some digging.

Reply:
Yeah it _should_ dynamically determine what languages the page is available in, but it hardcodes an arbitrary list here. Let's remove this entire file, and use the infogami native one.